README.md

```diff
@@ -1,80 +1,90 @@
 [![Build Status](https://travis-ci.org/k8sp/sextant.svg?branch=master)](https://travis-ci.org/k8sp/sextant.svg?branch=master)
 
-# sextant
+#Sextant
 <img src="logo/Sextant.png" width="250">
 
-sextant 提供了可以通过PXE全自动化安装初始化一个CoreOS+kubernetes集群。
+Sextant provides a tool that initialize a cluster with CoreOS and Kubernetes crossing PXE technology.
 
-## 环境准备
-bootstrapper需要运行在一台服务器上(以下称bootstrapper server)，满足以下的几个要求：
+# Enviroment Prepare
+Bootstrapper will running on a machine(bootstrapper server), which need to meet the following requirements
 
-1. 待初始化的kubernetes机器需要和bootstrapper server保持网络连通
-1. bootstrapper server是一台安装有docker daemon(***1.11以上版本***)的Linux服务器
-1. 拥有bootstrapper server的root权限
-1. 配置bootstrapper server的/etc/hosts文件，增加hostname的解析：```127.0.0.1  bootstrapper```
+1. The kubernetes machine to be initialized need to be connected with bootstrapper server.
+2. Bootstrapper server is a linux server with `docker daemon`(more than 1.11 version) installed.
+3. Root permissions with bootstrapper server.
 
-## 初始化配置和准备bootstrapper需要的镜像文件
-***在能访问互联网的一台机器上完成下面的准备环境，配置，创建Docker镜像的步骤***
-* 注：如果bootstrapper机器没有互联网访问，可以事先准备好/bsroot目录然后上传到bootstrapper server
+# Initial configuration and prepare image files bootstrapper needs.
+
+Complete the flowing steps that the enviromental preparation, configuration and build docker images.
+    - If no internet access, you can upload the prepared `./bsroot` directory to bootstrapper server.
+
+After getting the sextant code, initial the cluster planning, edit `cloud-config-server/template/cluster-desc.sample.yaml` and then build bootstrapper to the `./bsroot` directory.
```

Comment:
***The following steps will prepare the environment, generate configurations and build docker images.***
* if there's no internet access on the bootstrapper server, you can copy the pre-donwloaded `/bsroot` directory to it.

After getting the sextant code, you need to plan the cluster installation details by editing `cloud-config-server/template/cluster-desc.sample.yaml`. Then build bootstrapper to the `./bsroot` directory.